src/effects/presets/effectchainpresetmanager.cpp

```diff
@@ -123,6 +123,12 @@ void EffectChainPresetManager::importPreset() {
         EffectChainPresetPointer pPreset(
                 new EffectChainPreset(doc.documentElement()));
         if (!pPreset->isEmpty() && !pPreset->name().isEmpty()) {
+            // Don't allow '---' because that's the name of the internal empty preset
+            if (pPreset->name() == kNoEffectString) {
+                pPreset->setName(pPreset->name() +
+                        QLatin1String(" (") + tr("imported") + QLatin1String(")"));
```

Comment:
Since + calls QStringBuilder converting Laitn1 to utf16, it is more effective to use QStringLiteral here.

Reply:
You may also fix the code below.

Reply:
done

Reply:
> Since + calls QStringBuilder converting Laitn1 to utf16, it is more effective to use QStringLiteral here.

Could you please share the page where you got that from? Didn't find it (explicitly) in the QString doc